### ISSUE TITLE
OSC issue

### ISSUE THREAD

**Issue by thespica93** [xLightsSequencer/xLights#2210]
NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**
I watched videos on how to setup OSC in Xschedule. I have control back and forth between xschedule and the Ipad using OSC, however nothing is responding.  I have checked the logs in xschedule and I can see the button presses but they just say "Invalid OSC Command". I am not sure how to resolve this, as I think I have everything configured properly.

I have larger plans for this, just trying to test on a smaller scale. 

**To Reproduce**
1) Created a simple test sequence and imported it into xschedule
2) Created a simple TouchOSC file with a single button
3) Created an event in xschedule with OSC, and set to "stop all now"
4) Configured OSC in Xschedule for proper send and receive ports
5) Configured the Ipad with the Host IP address, and the correct ports

**Expected behavior**
I should be able to press the red button on the ipad and have the playlist that is playing to stop. 

**Screenshots**
This is the cover of xschedule to show that the Ipad is populating as a controller
![Xlights Dashboard](https://user-images.githubusercontent.com/42631192/92501366-72363900-f1cc-11ea-812e-79910591da0c.PNG)

This is the OSC Config in Xschedule with the Ports for incoming and outgoing configured. 
![OSC Config](https://user-images.githubusercontent.com/42631192/92501394-7bbfa100-f1cc-11ea-9806-7a2c85f1d88b.PNG)

This is the events page in xschedule set for OSC and with the path of the button
![Events](https://user-images.githubusercontent.com/42631192/92501410-80845500-f1cc-11ea-967f-bfc555e11140.PNG)

This is the TouchOSC editor showing the button with the path
![touchosc](https://user-images.githubusercontent.com/42631192/92501430-89752680-f1cc-11ea-8d81-b3b655ee7808.PNG)

This is the TouchOSC Ipad App Configuration page showing the ports and host IP configured.
![Touchosc ipad config](https://user-images.githubusercontent.com/42631192/92501441-8da14400-f1cc-11ea-8d43-972c945f7022.jpg)

**Versions (please complete the following information):**
 Windows 10
 - xLights version 2020.34

**Attachments**
This is the zip folder for the test folder I created
[test.zip](https://github.com/smeighan/xLights/files/5189541/test.zip)

Here is the log file from xschedule to show the "invalid OSC Commands"
[xSchedule_l4cpp.log](https://github.com/smeighan/xLights/files/5189567/xSchedule_l4cpp.log)

Thanks for all you guys do!

**Comment by keithsw1111** [comment]
I ran a test with syntien and it works fine.

I have added some logging to .36 which will log the content of the invalid packet ... maybe then we can work out what is wrong.

**Comment by keithsw1111** [comment]
Any update?

**Comment by thespica93** [comment]
You were correct. I was able to make it work using the same settings using syntien. But it does not work with the touchosc app.

**Comment by keithsw1111** [comment]
Hmmm ... can you capture me logs with current version using touchosc ... i am now logging packets i cant decode so i can try to work out why it failed.

**Comment by alentz44** [comment]
This is way late but after fighting with this for a while, I found that the path has to be at least 10 characters long to work. When I used less than 10 characters in the path I received the following xscheduler logged error: 
`9192 log_base [DEBUG] 2f 50 75 72 70 6c 65 70 31 00 00 00 2c 00 00 00     /Purple...,...`

When using 10 or more characters I received success. Note the number of objects or types of characters in the path didn't seem to make a difference (only the number of characters). Logged Success: 
`2023-07-24 16:04:42,660 9192 log_base [DEBUG] OSC Path: /Purple123.`

Versions used:
touch OSC: 1.2.2.175
XLights (I love this product! thanks Keith and all those that support it): 2023.11
